Common/Source/Comm/device.cpp

```diff
@@ -628,7 +654,6 @@ BOOL devInit() {
                 }
               }
             }
-            continue;
```

Comment:
are you sure ?

Reply:
yes, did  not work with it!

Reply:
mind, the loop is counting different now